### PR TITLE
Refactor webkit css file to put the classes in proper places #21

### PR DESCRIPTION
**Pull request by saadnoor**
This PR includes:

- Refactoring of webkit.css
- moving classes from webkit.css to component specific CSS

### CODE REVIEW

File: css/webkit.css [hunk 1]
```diff
@@ -6,136 +6,16 @@ body
...
-        width: calc( 100vw - 140px );
-        height: calc( 100vh - 250px );
-}
-
-.tileslist .BBjPanel {
-        width: calc( 100vw - 150px );
-        height: calc( 100vh - 250px );
+.gridholder {
+	width: calc( 100vw - 150px );
+	height: calc( 100vh - 250px );
```
Comment (trifiasco) [inline css/webkit.css:13]: I don't see any reference of this class anywhere in the code. As long as we are cleaning stuff, shouldn't we just remove things that are not being used?

File: css/webkit.css [hunk 2]
```diff
@@ -6,136 +6,16 @@ body
...
-
-.menutile.submenu:hover {
-    background-color: var(--bbj-color-primary-50);
-    color:white;
-}
-
-.tileselected {
-	background-color:var(--bbj-color-primary-50);
+	width: calc( 100vw - 150px ) !important;
+	height: calc( 100vh - 320px ) !important;
```
Comment (trifiasco) [inline css/webkit.css:18]: Same comment about no references

File: framework/PortalFrame/PortalFrame.css [hunk 3]
```diff
@@ -38,3 +38,27 @@
...
+
+@media screen and (max-width: 600px) {
+    .mainarea {
+		position:absolute;
+		left:0px;
+		width:100vw;
+		height:calc( 100vh - 44px );
+		min-height:calc( 100vh - 44px );
+	}	
+}
```
Comment (trifiasco) [inline framework/PortalFrame/PortalFrame.css:64]: This media query is no longer necessary. There's a media query to handle the `mainarea` for smaller screens, right above this code-block in the same file. 
Also the `height` and `width` property doesn't respect the presence(and state) of drawer menu which is properly handled in the other media query.
So either properly set the `height` and `width` property respecting the state of drawer menu or simply removing this block would suffice for now.

**Review by trifiasco — CHANGES_REQUESTED** [review]
Few remarks but other than that looks good.

File: framework/PortalFrame/PortalFrame.css (hunk 3 above)
Comment (saadnoor) [reply framework/PortalFrame/PortalFrame.css]: Thanks, removed.

**Pull request merged**